assets-src/styles/sass/50-core-components/_navigation.scss

```diff
@@ -108,13 +102,23 @@ li.top-nav-item + li.top-nav-item {
 	}
 
 	@include mq($max-width) {
-		margin-inline-end: rem(25);
-		padding-bottom: rem(5);
 		position: relative;
+		
+		&::after {
+			border: rem(1) solid $black;
+			border-radius: rem(8);
+			content: '';
+			display: block;
+			height: 100%;
+			pointer-events: none; 
+			position: absolute;
+			right: -1.5em;
```

Comment:
Again, I think it would make more sent to use the `em` mixin here for consistency.